.github/pull_request_template.md

```diff
@@ -1,41 +1,32 @@
 Fixes #XXXX.
 
-(Replace XXXX with the id of the issue fixed in this PR. Remove the above line if there is no corresponding
-issue. Don't reference the issue in the title of this pull-request.)
+_Replace XXXX with the id of the issue fixed in this PR. Remove this section if there is no corresponding issue. Don't reference the issue in the title of this pull-request._
```

Comment:
Since this is going to show up in the edit view, parens probably make more sense than underscores.